Unclear why, but we seem to need this to build on Python 3.14 on Windows.

diff --git a/gi/pygi-resulttuple.c b/gi/pygi-resulttuple.c
index 93170ea..27483fc 100644
--- a/gi/pygi-resulttuple.c
+++ b/gi/pygi-resulttuple.c
@@ -331,7 +331,7 @@ static void resulttuple_dealloc(PyObject *self) {
     Py_TYPE (self)->tp_free (self);
 
 done:
-    CPy_TRASHCAN_END (self)
+    CPy_TRASHCAN_END (self);
 }
 #endif
 
